wled00/cfg.cpp

```diff
@@ -52,6 +52,7 @@ bool deserializeConfig(JsonObject doc, bool fromFS) {
     CJSON(staticSubnet[i], nw_ins_0_sn[i]);
   }
 
+#ifndef WLED_DISABLE_AP
```

Comment:
@duncan we already explained that WLED_DISABLE_AP will not be accepted in the main WLED repo. Please keep WLED_DISABLE_AP parts out of this PR, and maintain them in your own fork.